### PR TITLE
Fixes #18144: Remove rsyslog dependency from 7.0

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18144

### CODE REVIEW

File: rudder-webapp/SOURCES/Makefile [hunk 1]
```diff
@@ -188,7 +187,6 @@ install: build initial-promises initial-ncf
 	install -m 640 slapd.conf $(DESTDIR)/opt/rudder/etc/openldap/slapd.conf
 	install -m 644 $(WEBAPPDIR)/ldap-inventory/inventory-repository/src/main/resources/ldap/inventory.schema $(DESTDIR)/opt/rudder/etc/openldap/schema/
 	install -m 644 $(CORE_RESOURCES)/ldap/rudder.schema $(DESTDIR)/opt/rudder/etc/openldap/schema/
-	install -m 644 rsyslog/rudder-slapd.conf $(DESTDIR)/etc/rsyslog.d/rudder-slapd.conf
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/Makefile:191]: don't we need to also remove that at postinst time ?

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
PR updated with a new commit

File: rudder-webapp/SOURCES/Makefile (hunk 1 above)
Comment (amousset) [reply rudder-webapp/SOURCES/Makefile]: This file is for slapd logs and not linked to reporting

Comment (amousset) [reply rudder-webapp/SOURCES/Makefile]: (it is not removed anymore in this PR)

**Review by peckpeck — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**